### ISSUE TITLE
HoverProvider from DB

### ISSUE THREAD

**Issue by Danieleeee** [mickeypearce/oradew-vscode#16]
Hi,
is possible to have HoverProvider form DB objects?
Like columns, tables, packages,....
Thanks.

**Comment by yevon** [comment]
For this kind of feature you could install the extension plsql-language, but I tested it and it doesn't recognise oradew project structure. I will look into it, as this extension in cojunction to oradew is awesome.

**Comment by mickeypearce** [comment]
Agreed! It has been shipped together with Oradew not long ago as a dependency extension but I then removed it and let the user decide which language extension to use. `plsql-language` also works beautifully with "Go to object" if an object is saved locally in structure (CTRL+click on object in code) etc,...

**Comment by yevon** [comment]
Mmm I will double check, last test I did it with it didn't work for me.

**Comment by yevon** [comment]
Is plsql-language mantained? I wee that last commit was two years ago? and pending pull requests.

**Comment by yevon** [comment]
I just tried to execute the plsql-language repository without success, already installed dependencies but...:

![image](https://user-images.githubusercontent.com/121157/97627801-2a919800-1a2c-11eb-9da0-aaee6556ab71.png)

**Comment by mickeypearce** [comment]
> Is plsql-language mantained? I wee that last commit was two years ago? and pending pull requests.

It seems so. My impression is that the owner is still present (checks issues, etc). just not very active, but maybe I am wrong.

**Comment by mickeypearce** [comment]
I checked the other his extension (local-history) and he was active in the last month ...

**Comment by yevon** [comment]
Do you have this extension working with oradew? I can't manage to get ir working, do I have to modify some settings? I get code highlighting right away, but no code completion or symbols detected.

**Comment by mickeypearce** [comment]
By symbols you mean Outline panel (Ctrl+Shift+o)? Idk it works for me, occasionally it breaks if I have some unusual code...

"Go To Definition" (Ctrl+click) also works, Peek definition, function specification completion... 

Proper code completion with DB connection isn't implemented as I know..

In settings I have:
```json
"files.associations": {
    "*.sql": "plsql"
  }
```
thats all.

**Comment by yevon** [comment]
I have this configuration as yours and I've never got this extension working  :(

**Comment by yevon** [comment]
I just discovered the issue, it was that my packages have the word EDITIONABLE, and the extension is not working with it.
